### ISSUE TITLE
Mario Kart GP: Cannot exit Test menu regression

### ISSUE THREAD

**Issue by legend800** [teknogods/teknoparrot-issues#110]
Using today's build, I cannot exit the test menu. For me this is a required step in using the Bananapassport - I can never get it to load unless I enter/exit the test menu. 

It always locks up here:
![2020-03-30 23_36_37-mkart3 (Not Responding)](https://user-images.githubusercontent.com/20983844/77995964-9a4a7380-72e1-11ea-938c-609b9f4e6779.png)

**Comment by ghost** [comment]
yes and when i use the teknoparrot 1.0.0.67 et teknoparrotui 1.0.0.305 work well

**Comment by ReaverTeknoGods** [comment]
Could you be more SPECIFIC. What controller, what input method(direct input / xinput) and maybe a video? These bug reports are terrible.

**Comment by ghost** [comment]
i use xbox360 controle , xinput , windows 10 pro 1909 , gforce gtx 745 , 6 go of memoiry , ssd 1to and manu hdd 2.5 , i use system for fix sega rally 3 and after remove the fix sega rally 3 and same probleme (wma 3)

**Comment by ghost** [comment]
when i use the version 1.0.0.305 i show correctly the squarre with item , the opponent are corectly in you bow  , and the slider work well , but for the version 1.0.0.317 , the opponent car shift , the slider don t show me the progression , and the item it s not display but before the last update work

**Comment by ghost** [comment]
for my problem , i found a solution , i delete my folder teknoparrot with the emulator , i create a new folder but not same name , i download the tp 1.0.0.040 and unzip in the folder , after i copie my UserProfiles i the new folder of teknoparrot and now mario kart gp work again , and for the problem of the graphic i also found , before when i launch the game work in 16/9 and it s for that the graphic are no good , now launch in 4/3 and the gfx it s ok
but i am not sure it s a good solution but for this time work

**Comment by legend800** [comment]
@ViRuS-MaN Don't spam this issue with unrelated problems. Also you can just EDIT your post to avoid pinging everyone watching the issue. 

@ReaverTeknoGods Let me know what other info I can provide. I don't see a log file to upload. 

Here's my settings:
![2020-04-07 20_30_12-](https://user-images.githubusercontent.com/20983844/78742807-d305d080-7911-11ea-815c-dbb71bf5b801.png)

Note: I've tried recreating my mkgp profile from scratch and it didn't help. 

The repro steps is just open and try to close the Test menu. Freezes immediately on this screen at the i/o check. This worked fine for me for over a year until I just tried the other day. Happens on both xinput or dinput. 
![2020-04-07 20_35_23-mkart3](https://user-images.githubusercontent.com/20983844/78743523-b074b700-7913-11ea-8e73-d769b6f7bac4.png)

**Comment by ghost** [comment]
i don t use windowed and autoacc off , but now it s ok i re install teknoparrot in another folder and work well

**Comment by ghost** [comment]
i just update teknoparrot but now it s not possible to connect with my bonanpassport
i have this 
![vxwp](https://user-images.githubusercontent.com/62475321/79685065-19263280-8236-11ea-99dd-effe66e387f9.png)
and when it ok for authentification check , i presse item bouton and f2 and i arrived in this windows but never quitte this windows
![mario](https://user-images.githubusercontent.com/62475321/79684992-b2087e00-8235-11ea-9031-f8637b3707b8.png)
